Formula/pywhat.rb

```diff
@@ -0,0 +1,60 @@
+class Pywhat < Formula
+  include Language::Python::Virtualenv
+
+  desc "What is that thing?"
```

Comment:
Can we use a slightly more informative description?

Reply:
Original description from: https://pypi.org/project/pywhat/
New description from: https://github.com/bee-san/pyWhat